c/floats-cbmc-regression/float6_true-unreach-call.c

```diff
@@ -23,7 +24,7 @@ int main()
   if(!(!(-2.0>=-1.0))) __VERIFIER_error();  
   
   // variables
-  float a, b, _a=a, _b=b;
```

Comment:
Was puzzled a bit that `_a` and `_b` are removed, but it's OK since they are unused.